clab/config.go

```diff
@@ -307,6 +308,21 @@ func (c *cLab) cmdInit(nodeCfg *NodeConfig, kind string) string {
 	return ""
 }
 
+// initialize environment variables
+func (c *cLab) envInit(nodeCfg *NodeConfig, kind string) map[string]string {
+	switch {
+	case nodeCfg.Env != nil:
+		return nodeCfg.Env
+
+	case c.Config.Topology.Kinds[kind].Env != nil:
+		return c.Config.Topology.Kinds[kind].Env
+
+	case c.Config.Topology.Defaults.Env != nil:
+		return c.Config.Topology.Defaults.Env
+	}
+	return nil
+}
+
```

Comment:
as discussed, maybe allow merging of envs. node > kind > defaults

Reply:
added in cb223b2